core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java

```diff
@@ -94,21 +103,25 @@ protected void runTest(
                 KryoNetNetwork kryoNetwork = new KryoNetNetwork(partyNetConf);
                 if (logPerformance) {
                   NetworkLoggingDecorator network = new NetworkLoggingDecorator(kryoNetwork);
-                  pls.add(network);
+                  aggregate.add(network);
```

Comment:
This will not work as intented, the network has two values 
- minBytesReceived
- maxBytesReceived

Which will be summed.